apps/api/src/services/configuration/configuration.service.ts

```diff
@@ -59,6 +60,9 @@ export class ConfigurationService {
       PROCESSOR_CONCURRENCY_PORTFOLIO_SNAPSHOT: num({
         default: DEFAULT_PROCESSOR_CONCURRENCY_PORTFOLIO_SNAPSHOT
       }),
+      PROCESSOR_PORTFOLIO_SNAPSHOT_COMPUTATION_TIMEOUT: num({
+        default: DEFAULT_PROCESSOR_PORTFOLIO_SNAPSHOT_COMPUTATION_TIMEOUT
+      }),
```

Comment:
I'm wondering if this should be `PROCESSOR_TIMEOUT_PORTFOLIO_SNAPSHOT` to be consistent with the other variables above?
```
PROCESSOR_CONCURRENCY_GATHER_ASSET_PROFILE
PROCESSOR_CONCURRENCY_GATHER_HISTORICAL_MARKET_DATA
PROCESSOR_CONCURRENCY_PORTFOLIO_SNAPSHOT
PROCESSOR_TIMEOUT_PORTFOLIO_SNAPSHOT
```

Thoughts?

Reply:
See here: https://github.com/ghostfolio/ghostfolio/pull/3913